apps/docusaurus/docs/concepts/txns-states.md

```diff
@@ -49,7 +49,7 @@ A signed transaction on the blockchain contains the following information:
 - **Sender public key**: The public authentication key that corresponds to the private authentication key used to sign the transaction.
 - **Payload**: Indicates an action or set of actions Alice's behalf. In the case this is a Move function, it directly calls into Move bytecode on the chain. Alternatively, it may be Move bytecode peer-to-peer [Move script](../reference/glossary.md#transaction-or-move-script). It also contains a list of inputs to the function or script. For this example, it is a function call to transfer an amount of Aptos Coins from Alice account to Bob's account, where Alice's account is implied by sending the transaction and Bob's account and the amount are specified as transaction inputs.
 - [**Gas unit price**](../reference/glossary.md#gas-unit-price): The amount the sender is willing to pay per unit of gas, to execute the transaction. This is represented as Octa or units of 10<sup>-8</sup> utility tokens.
-- [**Maximum gas amount**](../reference/glossary.md#maximum-gas-amount): The maximum gas amount in Aptos utility tokens the sender is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented as Octas or units of 10<sup>-8</sup> Aptos utility tokens.
+- [**Maximum gas amount**](../reference/glossary.md#maximum-gas-amount): The maximum gas amount in Aptos utility tokens the sender is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented in gas units, not Octas (i.e., 10<sup>-8</sup> Aptos utility tokens).
```

Comment:
Hmm... looks more more fixes are needed here

> The maximum gas amount in Aptos utility tokens

The maximum amount is measured in gas units

Might also help to clarify that the final fee is gas used * gas unit price

Reply:
I added the clarification.